src/shared/utils.ts

```diff
@@ -74,6 +79,9 @@ export const markdownFieldCharacterLimit = 50000;
 
 export const relTags = "noopener nofollow";
 
+let customEmojis: EmojiMartCategory[] = [];
+export let customEmojisLookup: Map<string, CustomEmojiView> = new Map<string, CustomEmojiView>();
```

Comment:
Much better. If we end up getting some weird double calls on this thing tho, we might have to set up a singleton service like was done for UserService and WebsocketService, that could hold some other globals as well.